src/core/js/a11y.js

```diff
@@ -380,7 +380,7 @@ class A11y extends Backbone.Controller {
 
     // check children by walking the tree
     let $found = this._findFirstForwardDescendant($element, iterator);
-    if ($found && $found.length) {
+    if ($found.length) {
```

Comment:
Won't these error out if $found is undefined?

Reply:
It always returns a jquery object.